src/drive/web/modules/drive/StyledApp.jsx

```diff
@@ -0,0 +1,7 @@
+import 'cozy-ui/transpiled/react/stylesheet.css'
+import 'cozy-sharing/dist/stylesheet.css'
+//eslint-disable-next-line
+import mainStyles from 'drive/styles/main.styl'
+
+const StyleApp = ({ children }) => children
+export default StyleApp
```

Comment:
Do we really need the component here? I would think the imports are enough no?

Reply:
Yep, but I plan to have a centralized way to handle the bootstrap of the public / private app. It'll be in this file (after a rename) ;)